src/Support/SupportData.php

```diff
@@ -0,0 +1,930 @@
+<?php
+/**
+ * Class to prepare and send support data to insights server.
+ *
+ * @package AmpProject\AmpWP
+ */
+
+namespace AmpProject\AmpWP\Support;
+
+use WP_Error;
+use AmpProject\AmpWP\QueryVar;
+
+/**
+ * Class SupportData
+ * To prepare and send support data to insights server.
```

Comment:
This class and other classes added here should be marked as `@internal`.

```suggestion
 * To prepare and send support data to insights server.
 *
 * @internal
```